gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaAvroJobStatusMonitor.java

```diff
@@ -166,6 +171,16 @@ public GobblinTrackingEvent deserializeEvent(DecodeableKafkaRecord<byte[],byte[]
       case TimingEvent.JOB_COMPLETION_PERCENTAGE:
         properties.put(TimingEvent.JOB_LAST_PROGRESS_EVENT_TIME, properties.getProperty(TimingEvent.METADATA_END_TIME));
         break;
+      case JobEvent.WORK_UNITS_CREATED:
+        Long numWorkUnits = Long.parseLong(properties.getProperty(CountEventBuilder.COUNT_KEY));
```

Comment:
properties.getProperty(CountEventBuilder.COUNT_KEY) is telling how many workunits are created?
It could have been saved with a more self-explanatory name inside the workunit properties

Reply:
When using a CountEvent that is the default name of the key used to store the count quantity if you see [here](https://jarvis.corp.linkedin.com/codesearch/result/?name=CountEventBuilder.java&path=gobblin-elr%2Fgobblin-metrics-libs%2Fgobblin-metrics-base%2Fsrc%2Fmain%2Fjava%2Forg%2Fapache%2Fgobblin%2Fmetrics%2Fevent&reponame=linkedin%2Fgobblin-elr#59). However, I could reverse create the `CountEvent` from a `GobblinTrackingEvent` where I would be able to specify the key name by using the `fromEvent` [builder](https://jarvis.corp.linkedin.com/codesearch/result/?name=CountEventBuilder.java&path=gobblin-elr%2Fgobblin-metrics-libs%2Fgobblin-metrics-base%2Fsrc%2Fmain%2Fjava%2Forg%2Fapache%2Fgobblin%2Fmetrics%2Fevent&reponame=linkedin%2Fgobblin-elr#74). It would be a sort of roundabout way though.